html/host.html

```diff
@@ -3,66 +3,100 @@
     <meta charset="utf-8"/>
     <link href="/assets/stylesheets/main.css" rel="stylesheet">
     <link href="/assets/stylesheets/host.css" rel="stylesheet">
+    <script type="module">
+      import { 
+        fullActivityDescription, 
+        shortActivityDescription, 
+        libraryModelList, 
+        cclDiseaseActivityUrl 
+      } from '/assets/data/host.js';
```

Comment:
Please try to mimic the `import` style (i.e. the code layout) that is used elsewhere in the codebase.

Reply:
Will do in the future sure (got rid of this file, so no imports here).